src/services/findAllReferences.ts

```diff
@@ -1145,7 +1154,7 @@ namespace ts.FindAllReferences.Core {
         const { symbol } = importOrExport;
 
         if (importOrExport.kind === ImportExport.Import) {
-            if (!state.options.isForRename) {
+            if (!(state.options.isForRename && state.options.providePrefixAndSuffixTextForRename)) {

```

Comment:
>state.options.isForRename && state.options.providePrefixAndSuffixTextForRename [](start = 18, length = 78)

Would it be worthwhile to extract a function options->bool?

Reply:
To clarify, not to reduce duplication but to maintain consistency (e.g. when negating).

Reply:
extracted.